src/main/java/juja/microservices/users/entity/User.java

```diff
@@ -1,36 +1,56 @@
 package juja.microservices.users.entity;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
 import lombok.Data;
-import lombok.Getter;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
 
 /**
  * @author Denis Tantsev (dtantsev@gmail.com)
  * @author Olga Kulykova
+ * @author Vadim Dyachenko
  */
 
+@Entity
+@Table(name = "USERS")
 @Data
 @AllArgsConstructor
 public class User {
-    @JsonProperty("c_uuid")
-    private String uuid;
-    @JsonProperty("firstName")
+
+    @Id
+    @Column(name = "UUID", unique = true, nullable = false)
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    private UUID uuid;
```

Comment:
so if I understand right the type UUID and type String are compatible? We can give the UUID to json as a String?